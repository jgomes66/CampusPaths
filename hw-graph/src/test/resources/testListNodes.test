# Test List Nodes
# Testing to see if the graph can list all of its nodes

# Create a graph
CreateGraph newGraph

# Add nodes to graph
AddNode newGraph node1
AddNode newGraph node2
AddNode newGraph node3

# List all nodes in graph
ListNodes newGraph