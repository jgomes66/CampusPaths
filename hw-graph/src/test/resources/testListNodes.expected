# Test List Nodes
# Testing to see if the graph can list all of its nodes

# Create a graph
created graph newGraph

# Add nodes to graph
added node node1 to newGraph
added node node2 to newGraph
added node node3 to newGraph

# List all nodes in graph
newGraph contains: node1 node2 node3